configs/components/razor-server.json

```diff
@@ -1 +1 @@
-{"url": "git://github.com/puppetlabs/razor-server.git", "ref": "1.8.1"}
+{"url": "git://github.com/mwaggett/razor-server.git", "ref": "3b8c0d4836330cdaf05656eee368e3db9a8ca2b6"}
```

Comment:
This PR is actually blocked on the razor-server PR, right?

Reply:
yeah, I just added this for testing, will remove when I'm done.